lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -1873,6 +1877,21 @@ def air_loop_hvac_adjust_minimum_vav_damper_positions(air_loop_hvac)
       return true
     end
 
+    # Set the minimum zone ventilation efficiency to be 0.6
+
+    # Use the built-in EnergyPlus method when available
+    if air_loop_hvac.model.version > OpenStudio::VersionString.new('3.0.0')
+      air_loop_hvac.thermalZones.sort.each do |zone|
+        sizing_zone = zone.sizingZone
+        sizing_zone.setDesignMinimumZoneVentilationEfficiency(0.6)
+      end
+      OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.AirLoopHVAC', "For #{air_loop_hvac.name}: the multizone outdoor air calculation method was applied, and a minimum zone ventilation efficiency of 0.6 was specified using the EnergyPlus algorithm.")
+
+      return true
+    end
+
+    # Use the manual method otherwise
+
```

Comment:
I don't believe that this is a code requirement, we're currently doing that in `model_system_outdoor_air_sizing_vrp_method()` in `Prototype.SizingSystem.rb`. What's the reason for moving it here and below?

Reply:
It's not a code requirement, however, the previous approach, which mainly affects ComStock or others using the older prototypes, assumed that historic design practice for the pre-1980 and 1980-2004 templates was simply to not adjust minimum damper positions, which results in higher min OA requirements for those buildings. When the change was made to put the 0.6 minimum limit into `model_system_outdoor_air_sizing_vrp_method()`, I didn't notice because ComStock was still using OS 2.9.1, so that code never got called.  I think that moving the code into standard-specific methods should result in the previous behavior:

- For pre-1980 and 1980-2004, VAV systems get Standard62.1VentilationRateProcedure with no minimum zone ventilation efficiency. This likely means higher minimum OA for those systems.
- For 2004+
    - If using OS =< 3.0.0, minimum damper positions are manually adjusted to achieve a minimum zone ventilation efficiency of 0.6
    - If using OS > 3.0.0, minimum damper positions are set to achieve a minimum zone ventilation efficiency of 0.6 using the newly-available EnergyPlus built-in algorithm.
    - I have tested these two methods on a small number of buildings and confirmed that the E+ algorithm and the manual method create nearly identical results.

Reply:
Ok, I see, thanks for the additional information. So when we worked on that section of the code (in #1112 and #776) we missed that the DOE reference building should not be using a minimum efficiency. Knowing that I think that what you're proposing makes sense but I do see that the regression and performance tests failed. Do you mind if I have a look at why we get diffs before this gets merged? Is it urgent, I might not have time to look at it before next week.